rapstore-frontend/src/app/app.component.html

```diff
@@ -27,9 +27,9 @@ <h4>The friendly App Store for the Internet of Things.</h4>
                     </div>
 
                 </div>
-                <div class="col-sm-2">
+                <!--<div class="col-sm-2">
```

Comment:
Please re-enable the badge for the RAPstore version, since we want an indicator for the beta status of the project

Reply:
Then I propose to just leave it as "BETA". What do you think?

Reply:
That is ok for me